backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/database/repos/AddressRepository.kt

```diff
@@ -2,9 +2,9 @@ package xyz.elitese.ehrenamtskarte.database.repos
 
 import xyz.elitese.ehrenamtskarte.database.AddressEntity
 import xyz.elitese.ehrenamtskarte.database.Addresses
-import xyz.elitese.ehrenamtskarte.database.associateWithKeys
+import xyz.elitese.ehrenamtskarte.database.sortByKeys
 
 object AddressRepository {
     fun findByIds(ids: List<Int>) =
-        AddressEntity.find { Addresses.id inList ids }.associateWithKeys({ it.id.value }, ids)
+        AddressEntity.find { Addresses.id inList ids }.sortByKeys({ it.id.value }, ids)
```

Comment:
How does this lead to sorting by location or is this done in frontend?

Reply:
It doesn't. We only renamed `associateWithKeys` to `sortByKeys`